karma.config.js

```diff
@@ -26,10 +26,14 @@ module.exports = function(config) {
 
   const karmaConfig = {
     browsers: ['ChromeHeadless'],
-    files: [{ pattern: testFilesPattern, watched: true, type: 'module' }],
+    files: [
+      'test/__bootstrap__/expect.assertions.js',
```

Comment:
☝️ Should this be a pattern too and not a hard coded path (thinking for future bootstrap files added)

Reply:
Good call